inttest/footloose-alpine/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.13
 
-RUN apk add openrc openssh-server bash busybox-initscripts coreutils findutils iptables curl haproxy
+RUN apk add openrc openssh-server bash busybox-initscripts coreutils findutils iptables curl haproxy openssh-client
```

Comment:
Do we really need the ssh-client packages?

Reply:
not anymore, I used it before I add volumes